scripts/host-svc-check

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+import os
+import time
+
+host = os.getenv('RHMAP_ROUTER_DNS', 'localhost')
+nagios_cmd_file = os.getenv('NAGIOS_CMD_FILE', '/var/spool/nagios/cmd/nagios.cmd')
+
+
+def force_service_checks():
+    print "Forcing service checks on '%s' ..." % (host)
+
+    with open(nagios_cmd_file, "a") as f:
+        cmd = "[%s] SCHEDULE_HOST_SVC_CHECKS;%s;1110741500\n" % (int(time.time()),host)
```

Comment:
Scheduled start time, doesn't actually matter what it is I'd say as long as it's before now since we are forcing it to do the check. https://old.nagios.org/developerinfo/externalcommands/commandinfo.php?command_id=122.

I'll add that link as a comment.